media-tv/kodi/kodi-9999.ebuild

```diff
@@ -234,28 +234,27 @@ src_configure() {
 		$(use_enable X x11)
 }
 
-src_compile() {
-	emake V=1
```

Comment:
I just pushed re-adding src_compile - that was a very interesting conversation. Thanks!